packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -120,23 +117,26 @@ class AppBar extends StatelessWidget {
   /// The following elevations have defined shadows: 1, 2, 3, 4, 6, 8, 9, 12, 16, 24
   final int elevation;
 
-  /// The color to use for the app bar's material. This generally should be set
-  /// in tandem with [brightness].
+  /// The color to use for the app bar's material. Typically this should be set
+  /// along with [brightness], [iconTheme], [textTheme].
   ///
   /// Defaults to [ThemeData.primaryColor].
   final Color backgroundColor;
 
-  /// The brightness of the app bar's material. This generally should be set in
-  /// tandem with [backgroundColor].
+  /// The brightness of the app bar's material. Typically this should be set along
+  /// with [backgroundColor], [iconTheme], [textTheme].
   ///
   /// Defaults to [ThemeData.brightness].
```

Comment:
Yes that makes sense since the backgroundColor default to the theme's primaryColor.